### DISCUSSION TITLE
Log out all sessions

### DISCUSSION THREAD

**Discussion by Kradre** [plausible/analytics#843] [Self-Hosted Support] [answered]
Have a question - how can I end all sessions of specific user? There's a little situation - I've created an account on self-hosted instance of plausible and gave access to other people for use. After some time, I've decided to change password so they'd ask me for access again, but unfortunately the change of password didn't force them to log out from account.

**Comment by ukutaht** [comment, marked as answer]
There's no way to do this currently but I think when a password changes we should log every session out automatically.

**Reply by Kradre** [reply]
Even if the password change was made through Elixir's console? Or there's other way to change it with refresh?
The command in elixir what I was using:
`Plausible.Repo.get_by(Plausible.Auth.User, email: "email@here.com") |> Plausible.Auth.User.set_password("passwordhere") |> Plausible.Repo.update
`

**Reply by ukutaht** [reply]
No, our sessions are not persisted and there is currently no way to end a session from the console
